apps/studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackWidget.tsx

```diff
@@ -158,33 +236,33 @@ export const FeedbackWidget = ({ onClose }: FeedbackWidgetProps) => {
     <ThanksMessage onClose={onClose} />
   ) : (
     <>
-      <div>
+      <>
         <div className="px-5 pb-4">
           <TextArea_Shadcn_
-            placeholder="It would be great if..."
-            rows={5}
+            placeholder="My idea for improving Supabase is..."
```

Comment:
More indicative placeholder text. More clearly marks this textarea as being for product feedback.